src/plugins/kibana_usage_collection/server/collectors/application_usage/README.md

```diff
@@ -74,11 +75,11 @@ The prop `viewId` is used as a unique identifier for your plugin. The Applicatio
 
 If you have a custom use case not provided by the Application Usage helpers you can use the `usageCollection.applicationUsageTracker` public api directly.
 
-To start tracking a view: `applicationUsageTracker.trackApplicationViewUsage(viewId)`
-Calling this method will marks the specified `viewId` as active. applicationUsageTracker will start tracking clicks and screen minutes for the view.
+To start tracking a view, call `applicationUsageTracker.trackApplicationViewUsage(viewId)` with an id for the view.
```

Comment:
Some more enhancements

Reply:
This is supposed to be removed in #92670

Reply:
👍